ENIGMAsystem/SHELL/Graphics_Systems/OpenGLES3/Makefile

```
@@ -1,2 +1,9 @@
 include Graphics_Systems/OpenGLES/Makefile
-SOURCES += $(wildcard Graphics_Systems/OpenGLES3/*.cpp)
+
+override CXXFLAGS += -I$(ENIGMA_ROOT)/android/external/gles3/include
+override CFLAGS += -I$(ENIGMA_ROOT)/android/external/gles3/include 
+SOURCES += $(wildcard Graphics_Systems/OpenGLES3/*.cpp) $(ENIGMA_ROOT)/android/external/gles3/src/gles2.c
+
+ifeq ($(TARGET-PLATFORM), Android)
+	ANDROID_LDLIBS += -lGLESv3
+endif
```

Shouldn't this be moved too?
```suggestion
ifeq ($(TARGET-PLATFORM), Android)
	override CXXFLAGS += -I$(ENIGMA_ROOT)/android/external/gles3/include
	override CFLAGS += -I$(ENIGMA_ROOT)/android/external/gles3/include 
	SOURCES += $(wildcard Graphics_Systems/OpenGLES3/*.cpp) $(ENIGMA_ROOT)/android/external/gles3/src/gles2.c
	ANDROID_LDLIBS += -lGLESv3
endif
```